ddtrace/appsec/_iast/_ast/ast_patching.py

```diff
@@ -120,6 +120,8 @@ def _remove_flask_run(text):  # type (str) -> str
     a new instance.
     """
     flask_instance_name = re.search(_FLASK_INSTANCE_REGEXP, text)
+    if not flask_instance_name:
+        return text
```

Comment:
is there any test that verify this fix?

Reply:
I added that to avoid an exception if we try to patch a file that doesn't contain app.run
Otherwise, next line will fail and we don't want that. It's not enough to test groups, as flask_instance_name can be None